qiskit/pulse/instructions/acquire.py

```diff
@@ -40,18 +41,24 @@ class Acquire(Instruction):
 
     def __init__(
         self,
-        duration: int | ParameterExpression,
-        channel: AcquireChannel,
-        mem_slot: MemorySlot | None = None,
-        reg_slot: RegisterSlot | None = None,
-        kernel: Kernel | None = None,
+        duration: Union[int, ParameterExpression],
+        *,
+        qubit: Optional[model.Qubit] = None,
```

Comment:
```suggestion
        qubit: model.Qubit | None = None,
```
Since we have `from __future__ import annotations`, you don't need to import Union and Optional from typing module. This change was made by https://github.com/Qiskit/qiskit/pull/8268 with considerable effort, and we don't need to switch back to the old convention. Same for others.